pkg/kube/operator/operator.go

```diff
@@ -46,3 +51,75 @@ func NewManager(ctx context.Context, config *config.Config, cfg *rest.Config, op
 
 	return mgr, nil
 }
+
+// ApplyCRDs applies a collection of CRDs into the cluster
+func ApplyCRDs(config *rest.Config) error {
+	exClient, err := extv1client.NewForConfig(config)
+	if err != nil {
+		return errors.Wrap(err, "Could not get kube client")
+	}
+	err = crd.ApplyCRD(
```

Comment:
This repetition looks like we could put the values in a `[]resource{{name: ..., kind: ...}, {}}` and loop over that, but then it's probably not less code. But if `ApplyCRD` or `WaitForCRDReady` ever change, we only have to fix one invocation.
Well, I have no strong preference here.

Reply:
It's better

Reply:
done